components/Onboarding/ConnectViaWallet/ConnectViaWalletTableViewItems.tsx

```
@@ -79,17 +77,32 @@ export function getConnectViaWalletInstalledWalletTableViewItem(args: {
+    useEffect(() => {
+      thirdwebActiveWalletRef.current = thirdwebActiveWallet;
+    }, [thirdwebActiveWallet]);
+
     const setThirdwebActiveWallet = useSetThirdwebActiveWallet();
 
+    const disconnectActiveThirdweb = useCallback(async () => {
+      if (!thirdwebActiveWalletRef.current) return;
+      disconnectThirdweb(thirdwebActiveWalletRef.current);
+      // Wait for the disconnect to complete
+      while (!!thirdwebActiveWalletRef.current) {
+        await new Promise((r) => setTimeout(r, 100));
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)